ios/Classes/AdmobBanner.swift

```diff
@@ -111,7 +111,10 @@ extension AdmobBanner : GADBannerViewDelegate {
     }
     
     func adView(_ bannerView: GADBannerView, didFailToReceiveAdWithError error: GADRequestError) {
-         channel.invokeMethod("failedToLoad", arguments: error)
+        channel.invokeMethod("failedToLoad", arguments: [
+            "errorCode": error.code,
+            "error": error.localizedDescription
+        ])
```

Comment:
This is what caused the banner crash. Dart expects a `Map`, this is more like it.